### ISSUE TITLE
Make the images list table header row sticky

### ISSUE THREAD

**Issue by gunamata** [rancher-sandbox/rancher-desktop#2611]
### Problem Description

When there's a long list of images, and if I scroll down through the list and if I want to select/deselect all images, I need to scroll all the way to the top of the table to do so.. Plus, with long images list, after scrolling down a bit, I lose track of which cell corresponds to which column.. I know there are not many columns in the table but there still is scope to improve the user experience here.

### Proposed Solution

Make the images list table header row sticky.

**Comment by kradchenko** [comment]
Is this issue still relevant? I would like to work on this.

**Comment by jandubois** [comment]
@kradchenko Yes, the issue is still relevant, but I believe somebody at SUSE is already looking into it.

Is there maybe something else you would like to work on?

**Comment by rak-phillip** [comment]
@kradchenko we could use some help updating our images table so that it displays a more friendly message when there's no content. I think this would be a good first issue if you're interested 🙂

https://github.com/rancher-sandbox/rancher-desktop/issues/1466
